### DISCUSSION TITLE
upgrade from next 9.4.4 to 10 - svg file import getting pass.get(...) error

### DISCUSSION THREAD

**Discussion by etaylib** [vercel/next.js#18406] [Help]
We decided to jump on and try version 10, but after upgrading to it, I get new errors.
I'm not sure if they are created directly due to a bug in next or the way the code is built, so I'm trying here before opening a bug.

**the error I get when building:**
![image](https://user-images.githubusercontent.com/17691874/97477778-b6d28b00-1958-11eb-9c72-9804e967c638.png)

**this is the imported file:**
![image](https://user-images.githubusercontent.com/17691874/97477786-bafea880-1958-11eb-81eb-b22378268340.png)

I'm pretty sure it's the way we **import SVG files**, after removing the import it worked, but then I got more errors on more similar issues.

I assume we can change it all but it all over the code, but it will be a bit time consuming, so do you know about any other way so I can keep the current code? maybe a missing babel setting or the next config?

using node v10.16.3

Thanks!

**Comment by timneutkens** [comment]
Looks similar to #18134, seems related to the new React JSX transform.
